bittensor/commands/root.py

```diff
@@ -283,7 +282,7 @@ def run(cli: "bittensor.cli"):
     def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         r"""Set weights for root network."""
         wallet = bittensor.wallet(config=cli.config)
-        subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
+        # subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
```

Comment:
let's delete this. I checked, it just makes unnecessary code and doesn't have any effect.
the same below